systems/subscriber/subscriber-registry/pkg/db/subscriber_repo_test.go

```diff
@@ -0,0 +1,124 @@
+package db
```

Comment:
we can add more tests for the other repo methods.

Reply:
Sure